src/main/java/FindCommand.java

```diff
@@ -0,0 +1,25 @@
+import java.util.ArrayList;
+
+public class FindCommand extends Command {
+
+    String keyword = "";
+
+    public void setKeyword(String keyword) {
```

Comment:
Perhaps "keyword" could be passed into the object when it is constructed, instead of having to create another method to set keyword under the object!